src/pipert2/core/base/logger.py

```diff
@@ -4,13 +4,25 @@
 class PipeLogger(ABC):
```

Comment:
why don't use the regular python logging library?

Reply:
add a bit of documentation - mainly how does this work with different threads and processes